docs/self-managed/operational-guides/update-guide/840-to-850.md

```diff
@@ -6,6 +6,14 @@ description: "Review which adjustments must be made to migrate from Camunda 8.4.
 
 The following sections explain which adjustments must be made to migrate from Camunda 8.4.x to 8.5.x for each component.
 
+## Helm chart
+
+:::caution Breaking changes
```

Comment:
Given these breaking changes, should this be noted elsewhere in the Helm documentation as well?

Reply:
I see that we have it in [upgrade.md](https://github.com/camunda/camunda-docs/pull/3490/files#diff-04b8fde9378d702f563a03ad401099a8ba243547b5b2cd8c13258f00c7f429f5), but wasn't sure if you felt we should note this in the release content.